python/VERSION

```diff
@@ -1 +1 @@
-0.11.0
+0.11.1
```

Comment:
should keep a newline at the end of file?

Reply:
Thanks @yuzisun  for fixing these.